src/concrete/ml/common/utils.py

```diff
@@ -37,7 +37,7 @@
 # Use new VL with .simulate() by default once CP's multi-parameter/precision bug is fixed
 # TODO: https://github.com/zama-ai/concrete-ml-internal/issues/3856
 # Indicate if the old simulation method should be used when simulating FHE executions
-USE_OLD_VL = True
+USE_OLD_VL = False
```

Comment:
this is a test, we'll see if we change the default (ie, does it break some tests or does it change build execution times)